pkg/instancetype/upgrade_test.go

```diff
@@ -54,26 +52,18 @@ var _ = Describe("ControllerRevision upgrades", func() {
 
 		vm *virtv1.VirtualMachine
 
-		virtClient  *kubecli.MockKubevirtClient
-		vmInterface *kubecli.MockVirtualMachineInterface
-		k8sClient   *k8sfake.Clientset
+		k8sClient *k8sfake.Clientset
 	)
 
 	BeforeEach(func() {
 		controllerrevisionInformer, _ := testutils.NewFakeInformerFor(&appsv1.ControllerRevision{})
 		controllerrevisionInformerStore := controllerrevisionInformer.GetStore()
 
-		ctrl := gomock.NewController(GinkgoT())
-		virtClient = kubecli.NewMockKubevirtClient(ctrl)
-		vmInterface = kubecli.NewMockVirtualMachineInterface(ctrl)
-		virtClient.EXPECT().VirtualMachine(metav1.NamespaceDefault).Return(vmInterface).AnyTimes()
-
 		k8sClient = k8sfake.NewSimpleClientset()
-		virtClient.EXPECT().AppsV1().Return(k8sClient.AppsV1()).AnyTimes()
 
 		methods = &InstancetypeMethods{
 			ControllerRevisionStore: controllerrevisionInformerStore,
-			Clientset:               virtClient,
+			Clientset:               k8sClient,
```

Comment:
Does this compile?

Reply:
@xpivarc no. i think i have to change to instancetype method in `instancetype.go`
`	Clientset                kubecli.KubevirtClient`

Reply:
Please make sure to run  the unit tests locally.

Reply:
@xpivarc which fakeclient should i use ? kubevirt's or k8s .